applications/RomApplication/tests/structural_static_test_files/LSPGROM/ProjectParametersLSPGROM.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Isn't this identical to the other one? Can't we unify them if so?

Reply:
Updated all tests to avoid repeating ProjectParameters and MDPAs.